src/pkg/bundle/deploy.go

```diff
@@ -110,62 +110,58 @@ func deployPackages(ctx context.Context, packagesToDeploy []types.Package, b *Bu
 			return err
 		}
 
-		opts := zarfTypes.ZarfPackageOptions{
-			PackageSource:      pkgTmp,
-			OptionalComponents: strings.Join(pkg.OptionalComponents, ","),
-			PublicKeyPath:      publicKeyPath,
-			SetVariables:       pkgVars,
-			Retries:            b.cfg.DeployOpts.Retries,
-		}
-
-		zarfDeployOpts := zarfTypes.ZarfDeployOptions{
-			ValuesOverridesMap: valuesOverrides,
-			Timeout:            config.HelmTimeout,
-		}
-
-		// Automatically confirm the package deployment
-		zarfConfig.CommonOptions.Confirm = true
-
-		source, err := sources.NewFromLocation(*b.cfg, pkg, opts, sha, nsOverrides)
```

Comment:
Known gap: need to address how this originally mapped a source to a package within a bundle. We may need similar logic to perform this before the zarf load package logic.

Reply:
resolved